src/averageAges.js

```diff
@@ -15,11 +23,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const listOfMen = people.filter(({ sex, died }) => {
+    return (sex === MAN) && (
+      century
+        ? Math.ceil(died / 100) === century
+        : true
+    );
+  });
+  const averageAge = listOfMen.reduce(
+    (acc, { born: manBorn, died: manDied }) => {
+      return acc + lifeTime(manBorn, manDied);
+    }, COUNT);
```

Comment:
why do u need a global variable?

```suggestion
    }, 0);
```